pkg/server/metrics/metrics.go

```diff
@@ -110,10 +112,21 @@ func newServerMetrics() *ServerMetrics {
 			Namespace: Namespace,
 			Subsystem: Subsystem,
 			Name:      "ready_backend_connections",
-			Help:      "Number of konnectivity agent connected to the proxy server",
+			Help:      "Number of konnectivity agent connected to the proxy server. DEPRECATED, please use ready_backend_connections_total",
 		},
 		[]string{},
 	)
+	totalBackendCount := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: Namespace,
+			Subsystem: Subsystem,
+			Name:      "ready_backend_connections_total",
```

Comment:
Final concern: the _total suffix here seems to contradict prometheus naming conventions (it implies a counter metric, not gauge).

Maybe just "ready_backends" or similar?